tool.go

```diff
@@ -480,7 +480,7 @@ func main() {
 		var root string
 
 		if len(args) > 1 {
-			cmdServe.Help()
+			cmdServe.HelpFunc()
```

Comment:
You're probably right.  I did some simple testing, but I in light of that, I don't think I was thorough enough. I'll close this PR, now that you're working on it as well.